plugins/inputs/smart/smart.go

```diff
@@ -87,6 +87,14 @@ var (
 		"199": "udma_crc_errors",
 	}
 
+	// There are some fields we're interested in which use the vendor specific device ids
+	// so we need to be able to match on name instead
+	deviceFieldNames = map[string]string{
+		"Percent_Lifetime_Remain": "endurance_remain_perc",
+		"Wear_Leveling_Count":     "endurance_wear_levelling",
+		"Media_Wearout_Indicator": "endurance_media_wearout",
```

Comment:
Can you please explain this name change? We tend to stick with the upstream names unless there is a good reason to change it

Reply:
Yep you're right, it's not needed here. Originally all the inputs were mapped to the same output name (as the way that endurance is indicated differs across manufacturers/models), but that was backed out in 177f6c1

I'll update the PR in a while to fix this

Reply:
PR updated (and tests re-run/fixed against the changes)

Reply:
great! Thank you so much